src/Schema/Attribute/AttributeDef.php

```diff
@@ -6,27 +6,49 @@
 
 class AttributeDef extends Item implements AttributeItem
 {
+    /**
+     * @var static|null
```

Comment:
here is `string` instead of `static|null`

Reply:
as above.